var/spack/repos/builtin/packages/geant4/package.py

```diff
@@ -0,0 +1,83 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+class Geant4(Package):
+    """Geant4 is a toolkit for the simulation of the passage of particles through matter.\
+        Its areas of application include high energy, nuclear and accelerator physics, \
+        as well as studies in medical and space science."""
```

Comment:
In Python, triple quotes signify a multiline comment. As such, the backslashes at the end of the lines aren't necessary. Instead, they create a single line comment that contains long stretches of whitespace. I believe that is what is causing the documentation tests to fail with messages like:

```
Warning, treated as error:
/home/travis/build/LLNL/spack/lib/spack/docs/package_list.rst:4770: WARNING: Block quote ends without a blank line; unexpected unindent.
```